plugins/migration/core/root.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"knative.dev/client-contrib/plugins/migration/pkg/command"
+	"knative.dev/client-contrib/plugins/migration/pkg/command/list"
+	"knative.dev/client-contrib/plugins/migration/pkg/command/migrate"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// migrationCmd represents the base command when called without any subcommands
+func NewMigrationCommand() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "kn\u00A0migration",
+		Short: "A plugin of kn client to migrate Knative services",
+		Long: `kn admin: a plugin of kn client to migrate Knative services.
```

Comment:
typo here "kn admin"

Reply:
Thanks for the review @zhanggbj !

Fixed! :)